lib/app/services/media_service/banuba_service.dart

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/services.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:ion/app/features/core/providers/env_provider.dart';
+import 'package:ion/app/services/logger/logger.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+import 'package:ve_sdk_flutter/features_config.dart';
+import 'package:ve_sdk_flutter/ve_sdk_flutter.dart';
+
+part 'banuba_service.g.dart';
+
+class BanubaService {
+  const BanubaService(this.env);
+  final Env env;
+
+  // For Photo Editor
+  static const methodInitPhotoEditor = 'initPhotoEditor';
+  static const methodStartPhotoEditor = 'startPhotoEditor';
+  static const argExportedPhotoFile = 'argExportedPhotoFilePath';
+
+  static const platformChannel = MethodChannel('banubaSdkChannel');
+
+  Future<void> _initPhotoEditor() async {
+    await platformChannel.invokeMethod(
+      methodInitPhotoEditor,
+      env.get<String>(EnvVariable.BANUBA_TOKEN),
+    );
+  }
+
+  Future<String> startPhotoEditor(String filePath) async {
```

Comment:
Possible rename the methods `startPhotoEditor`  and `openTrimmerScreen`  to `editPhoto` and `trimVideo` ?